psi4/driver/procrouting/response/scf_response.py

```diff
@@ -780,126 +780,84 @@ def tda_solver(e, n, g, m):
             names.add("DFT")
 
         for name in names:
+
+            def root_setter(prop_name, val):
+                wfn.set_variable(f"TD-{name} ROOT {i+1} {prop_name}", val)
+                wfn.set_variable(f"TD-{name} ROOT {target_h_count} (IN {x.irrep_ES}) {prop_name}", val)
+                wfn.set_variable(f"TD-{name} ROOT {i+1} ({x.irrep_ES}) {prop_name}", val)
+                wfn.set_variable(f"TD-{name} ROOT {i+1} {prop_name} - {x.irrep_trans} TRANSITION", val)
+
+            def transition_setter(prop_name, val):
+                wfn.set_variable(f"TD-{name} ROOT 0 -> ROOT {i+1} {prop_name}", val)
+                wfn.set_variable(f"TD-{name} ROOT 0 (IN {x.irrep_GS}) -> ROOT {target_h_count} (IN {x.irrep_ES}) {prop_name}", val)
+                wfn.set_variable(f"TD-{name} ROOT 0 ({x.irrep_GS}) -> ROOT {i+1} ({x.irrep_ES}) {prop_name}", val)
+                wfn.set_variable(f"TD-{name} ROOT 0 -> ROOT {i+1} {prop_name} - {x.irrep_trans} TRANSITION", val)
+
             # wfn.set_variable("TD-fctl ROOT m TOTAL ENERGY")               # P::e SCF
+            # wfn.set_variable("TD-fctl ROOT m (IN h) TOTAL ENERGY")           # P::e SCF
```

Comment:
there could possibly be some script or sphinx linking failure when like-qcvars get stacked in the glossary b/c of `m (IN h)` here and `m (IN i)` above `wfn.set_variable("ADC ROOT 0 (IN h) -> ROOT m (IN i) OSCILLATOR STRENGTH (LEN)")       # P::e ADC`

Reply:
That thought occurred to me as well, but alas, I didn't have a good way to hunt these down. If you have suggestions, I'd love to hear them.

Reply:
I think a docs build would throw up warnings that term collected from scanning code is missing in glossary. (It won't fail just now b/c we have warnings-into-errors off.) We can coordinate on docs build if you don't have a local one going.

Reply:
I do not, given how poorly the last attempt at a local docs build went. 😅